ingestion/src/metadata/ingestion/source/database/clickhouse/metadata.py

```diff
@@ -153,19 +153,20 @@ def get_schema_definition(
         Get the DDL statement or View Definition for a table
         """
         try:
-            if table_type in {TableType.View, TableType.MaterializedView}:
-                definition_fn = inspector.get_view_definition
-                schema_definition = definition_fn(table_name, schema_name)
-            else:
-                schema_definition = inspector.get_table_ddl(
-                    self.connection, table_name, schema_name
+            if self.source_config.includeDDL:
+                if table_type in {TableType.View, TableType.MaterializedView}:
```

Comment:
let's keep the code style constant if we do `if;return;return` let's do this everywhere and not switch between `if;return;return` and `if;else;return`

Reply:
will change the code style into `if;return;return`